src/s1/s1/static/js/Packages/S1/Workbench/Workbench.ts

```diff
@@ -294,7 +294,7 @@ var getMeeting = (proposal : RIProposal, process : RIS1Process) => {
         var proposalDecisionDate = AdhUtil.deepPluck(
             AdhProcess.getStateData(proposal.data[SIWorkflowAssignment.nick], proposalState), ["start_date"]);
 
-        return (processDecisionDate === proposalDecisionDate) ? "current" : "archive";
+        return (proposalDecisionDate <= processDecisionDate) ? "archive" : "current";
```

Comment:
Just so we do not forget it. IIRC, these values are strings, so comparison might not work as expected.